templates/index.html

```diff
@@ -5,18 +5,34 @@
     <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
   </head>
   <body>
+  <center>
```

Comment:
We may want to save styling for when we have all the content.

Reply:
Well now that the backend work is out of this ticket, maybe the styling is mostly what is left and thus we should include it?

Reply:
That was what I was going for. Didn't wanna overdo it since this will change so much. My long term plan is to use a `Google Material Design` library to make it super pretty.